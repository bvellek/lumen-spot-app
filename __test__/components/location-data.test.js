import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import LocationData from '../../public/js/components/location-data';

describe('<LocationData />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <LocationData />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
