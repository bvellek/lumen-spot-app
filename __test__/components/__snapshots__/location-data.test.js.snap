// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationData /> it renders correctly 1`] = `
<div
  className="info-container"
>
  <Connect(SunResultsContainer) />
  <Connect(WeatherResultsContainer) />
</div>
`;
